submissions/unabyband/oop_exercise/index.js

```diff
@@ -0,0 +1,87 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/unabyband/a-tiny-JS-world
+   Web app: https://unabyband.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+  constructor (species, name, gender, limbs, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.limbs = limbs;
```

Comment:
Introduce legs and hands and/or paws. In OOP-compliant manner.
Will be happy to see if you mastered OOP - that's the purpose of the module.

Reply:
Done!